survey-sdk/src/main/java/co/nimblehq/survey/sdk/entity/MetaEntity.kt

```diff
@@ -0,0 +1,13 @@
+package co.nimblehq.survey.sdk.entity
+
+import com.squareup.moshi.Json
+
+data class MetaEntity(
+    val page: Int,
+    val pages: Int,
+
+    @Json(name = "page_size")
+    val pageSize: Int,
+
+    val records: Int
```

Comment:
I'd recommend adding the annotation for all of the properties for consistency. Furthermore, in case you renaming the properties, it will not affect the parsing mechanism.

Reply:
@hoangnguyen92dn agree with this point